README.md

```diff
@@ -116,19 +116,22 @@ Now, login to the OpenShift cluster as user `developer` and create a build from
 ```bash
 # on host system
 $ oc login -u developer
-$ oc process -p PIPELINE_REPO=${PIPELINE_REPO} -p PIPELINE_BRANCH=${PIPELINE_BRANCH} -p REGISTRY_URL=${REGISTRY_URL} -p NAMESPACE=`oc project -q` -f seed-job/buildtemplate.yaml | oc create -f - 
+$ oc process -p PIPELINE_REPO=${PIPELINE_REPO} -p PIPELINE_BRANCH=${PIPELINE_BRANCH} -p REGISTRY_URL=${REGISTRY_URL} -p NAMESPACE=`oc project -q` -p FROM_ADDRESS=${FROM_ADDRESS} -p SMTP_SERVER=${SMTP_SERVER} -f seed-job/buildtemplate.yaml | oc create -f - 
 ```
```

Comment:
should not we have an option about authentication also whenever required?

Reply:
How do we test that when we don't have a server for it? Maybe we keep it for future contributors to help us with when they fall into that situation?